genvalidity-bytestring/src/Data/GenValidity/ByteString.hs

```diff
@@ -15,6 +15,32 @@ import qualified Data.ByteString.Internal as SB
 import qualified Data.ByteString.Lazy as LB
 import qualified Data.ByteString.Lazy.Internal as LB
 
+instance GenUnchecked SB.ByteString where
+    genUnchecked =
+        error $
+        unlines
+            [ "Data.GenValidity.ByteString.genUnchecked :: Strict.ByteString"
+            , "You probably do not want to use this."
+            , "You probably want to use 'genValid' instead."
+            , "See https://github.com/NorfairKing/validity/blob/master/docs/BYTESTRING.md"
+            ]
+    shrinkUnchecked =
+        error $
+        unlines
+            [ "Data.GenValidity.ByteString.shrinkUnchecked :: Strict.ByteString -> [Strict.ByteString]"
+            , "You probably do not want to use this."
+            , "You probably want to use 'shrinkValid' instead."
+            , "See https://github.com/NorfairKing/validity/blob/master/docs/BYTESTRING.md"
+            ]
```

Comment:
@lehins How's this?

Reply:
Looks great. Much better than it was before ;)